examples/bigcommerce/README.md

```diff
@@ -0,0 +1,198 @@
+# Demo
+
+[https://www.makeswift-examples-bigcommerce.vercel.app](https://www.makeswift-examples-bigcommerce.vercel.app)
```

Comment:
<img width="1034" alt="CleanShot 2022-08-12 at 11 04 08@2x" src="https://user-images.githubusercontent.com/20950876/184396562-27cbffec-803a-4c29-bd36-9d200fdc9f3d.png">
This link will work when this is deployed. I can't change the domain so that it works till then